eslint/babel-eslint-parser/test/non-regression.js

```diff
@@ -18,26 +18,23 @@ function verifyAndAssertMessagesWithSpecificESLint(
       node: true,
       es6: true,
     },
+    ...overrideConfig,
     parserOptions: {
       sourceType,
       ecmaFeatures: {
         globalReturn: true,
```

Comment:
Can we remove this? It's better to set it explicitly in the only test where we need it rather than having a default value.